wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/WorkspaceLinksGenerator.java

```diff
@@ -60,7 +60,6 @@ public WorkspaceLinksGenerator(
         LINK_REL_SELF,
         uriBuilder
             .clone()
-            .path(WorkspaceService.class)
```

Comment:
I'm OK with it if it fixes the issue. But it is still unclear why it works fine for [User API](https://github.com/eclipse/che/blob/6e9825c699d34eef6b8bf36fc0194ff871cbee84/wsmaster/che-core-api-user/src/main/java/org/eclipse/che/api/user/server/UserLinksInjector.java#L49) in the same way.

Reply:
-   ```serviceContext.getServiceUriBuilder()``` in WorkspaceLinksGenerator
-  ```serviceContext.getBaseUriBuilder()``` in UserLinksInjector

Reply:
I see. Thx.